mlir/include/mlir/Dialect/LLVMIR/Transforms/Passes.h

```diff
@@ -23,6 +23,10 @@ namespace LLVM {
 /// Create a pass to add DIScope to LLVMFuncOp that are missing it.
 std::unique_ptr<Pass> createDIScopeForLLVMFuncOpPass();
 
+struct DIScopeForLLVMFuncOpOptions;
+std::unique_ptr<Pass>
+createDIScopeForLLVMFuncOpPass(const DIScopeForLLVMFuncOpOptions &options);
+
```

Comment:
I think that removing the `let constructor = ` from the TableGen declaration for this pass will see the createXXX method auto-generated. Can you try this?

Reply:
Thanks, then the changes become more concise.